<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>821</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FishROS CAN Debuger(VCODE) fishros.org.cn</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QTableView" name="tableViewCANFrame"/>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>0</height>
         </size>
        </property>
        <property name="tabletTracking">
         <bool>true</bool>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>设备</string>
         </property>
        </widget>
        <widget class="QComboBox" name="commonBoxDevices">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>10</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>请选择设备端口号</string>
         </property>
         <item>
          <property name="text">
           <string>/dev/ttyUSB0</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>帧率</string>
         </property>
        </widget>
        <widget class="QComboBox" name="commonBoxRateSet">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>90</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>请选择设备端口号</string>
         </property>
         <item>
          <property name="text">
           <string>50</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>100</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>125</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>250</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>500</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1000</string>
          </property>
         </item>
        </widget>
        <widget class="QPushButton" name="pushButtonSetRate">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>90</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>设置</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBoxAutoScoll">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>250</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>自动滚动</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonClearScreen">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>250</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>清空屏幕</string>
         </property>
        </widget>
        <widget class="QComboBox" name="commonBoxMaxFrameCount">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>210</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>请选择设备端口号</string>
         </property>
         <item>
          <property name="text">
           <string>50</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>100</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10000</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>210</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>显示帧数</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>过滤  ID</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditFilterID">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>170</y>
           <width>181</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>00 00 00 00</string>
         </property>
        </widget>
        <widget class="QComboBox" name="commonBoxFilterID">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>130</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>请选择设备端口号</string>
         </property>
         <item>
          <property name="text">
           <string>接收全部</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>指定ID</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>指定  ID</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>操作</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonDeviceOpen">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>50</y>
           <width>81</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>打开</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonDeviceClose">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>50</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonDeviceFresh">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>10</y>
           <width>51</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>250</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>显示设置</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>290</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>数据速率</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelRecvRate">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>290</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>290</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>帧/秒</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>290</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>接收</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>320</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>320</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>帧/秒</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelSendRate">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>320</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>350</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>总线信息</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="labelBusErrorCount">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>350</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>350</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>错误计数</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>380</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>总线状态</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="labelBusStatus">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>380</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>700</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>600</height>
         </size>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>CAN发送</string>
         </attribute>
         <widget class="QTableView" name="tableViewSendCan">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>701</width>
            <height>221</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonAddSend">
          <property name="geometry">
           <rect>
            <x>610</x>
            <y>230</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonClearSend">
          <property name="geometry">
           <rect>
            <x>520</x>
            <y>230</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>清空</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>帮助</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>日志</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditSystemLog">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_about">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_fishros"/>
   </widget>
   <widget class="QMenu" name="menu_shop">
    <property name="title">
     <string>官方店铺</string>
    </property>
    <addaction name="action_shop"/>
   </widget>
   <addaction name="menu_about"/>
   <addaction name="menu_shop"/>
  </widget>
  <action name="action_fishros">
   <property name="text">
    <string>鱼香ROS</string>
   </property>
  </action>
  <action name="action_shop">
   <property name="text">
    <string>官方店铺</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
